custom_platforms.js

```diff
@@ -311,6 +311,14 @@ module.exports = {
                 ":has-text\\(",
                 ":contains\\(",
                 ":matches-css\\(",
+                ":matches-attr\\(",
+                ":matches-property\\(",
+                ":xpath\\(",
+                ":nth-ancestor\\(",
+                ":upward\\(",
+                ":remove\\(",
+                ":is\\(",
+                ":not\\(",
```

Comment:
both `:is()` and `:not()` can be standard pseudo-classes. should they be excluded as well?

Reply:
In fact, adding them was questionable and the only reason was that they are the part of the https://github.com/AdguardTeam/ExtendedCss. I wouldn't exclude them, what's your opinion?
I'll edit the description.